src/SignhostAPIClient/Rest/DataObjects/Signer.cs

```diff
@@ -37,6 +37,7 @@ public class Signer
 
 		public bool? SendSignConfirmation { get; set; }
 
+		[Obsolete("Use transaction.DaysToRemind instead")]
```

Comment:
@MrJoe 
It was supposed to be moved to the transaction object was it not?
So why signer[].DaysToRemind?